contracts/core/Modifier.sol

```diff
@@ -73,7 +74,13 @@ abstract contract Modifier is Module, IAvatar {
     */
 
     modifier moduleOnly() {
-        if (modules[msg.sender] == address(0)) revert NotAuthorized(msg.sender);
+        if (modules[msg.sender] == address(0)) {
```

Comment:
What if a relayer's address is enabled as a module? Then it can no longer relay messages for other accounts?

Not sure that this is a practical edge case that actually needs to be solved for.

Reply:
I don’t think this scenario is practical. If a relayer address is enabled, it should not relay to this particular contract as it would be like giving a blank check.